makefile

```diff
@@ -33,3 +33,10 @@ bin/dfmt: $(SRC)
 
 debug: $(SRC)
 	$(DC) $(DMD_DEBUG_FLAGS) $^ -ofbin/dfmt
+
+pkg: dmd
+	$(MAKE) -f makd/Makd.mak pkg
+
+clean:
+	$(RM) bin/dfmt
```

Comment:
You could remove the object files and test binary too...

Reply:
Since this project is not using dub for building, I think at some point it would be a good idea to migrate it to Makd, it will take care of all those details :)

Reply:
This project DOES dub for building and we actually need to maintain the dub-path as well.
for `dub fetch dfmt && dub run dfmt` to still work